src/app/shared/pages/order/order.component.ts

```diff
@@ -0,0 +1,75 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { Location } from '@angular/common';
+
+import { CartItem } from '../../../core/models/cart-item';
+import { OrderModel } from '../../models/order';
+
+import { CartService } from '../../../core/services/cart/cart.service';
+import { HttpDataService } from '../../../core/services/http-data/http-data.service';
+
+import { Payment } from '../../../shared/constants/payment';
+
+@Component({
+  selector: 'app-order',
+  templateUrl: './order.component.html',
+  styleUrls: ['./order.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrderComponent  implements OnInit {
+  payments = Payment;
+  order: OrderModel;
+  checkoutForm = this.formBuilder.group({
```

Comment:
checkoutForm only type here 
and formBuilder in ngOnInit

Reply:
fixed